### PR TITLE
Time ntp web gui

### PR DESCRIPTION
**Pull request by HuuPhuc25**
Create time_ntp.html for webs.

### CODE REVIEW

**Comment by hhdang6637** [comment]
@quoctri1

File: applications/web_server/html/time_ntp.html [hunk 1]
```diff
@@ -67,4 +80,60 @@ <h3>Time - NTP</h3>
...
+        txtTime = $('#txtTime').val();
+    }
+
+    e.preventDefault();
+
+    var formData = new FormData();
+    formData.append('enable_ntp', chkNtpEnable);
+    formData.append('ntp_server', txtNtpServer);
+    formData.append('date', txtDate);
+    formData.append('time', txtTime);
```
Comment (quoctri1) [inline applications/web_server/html/time_ntp.html]: Should we follow openvpn or adduser page to append data in FormData as a standard?

File: applications/web_server/src/time_ntp_js.cpp [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        if (get_post_data(request, data)) {
+            try {
+                MPFD::Parser POSTParser;
+
+                POSTParser.SetContentType(contentType);
+
+                POSTParser.AcceptSomeData(data.c_str(), data.size());
+
+                std::string enable_ntp = POSTParser.GetField("enable_ntp")->GetTextTypeContent();
+                printf("json_handle_time_ntp: enable_ntp: %s\n", enable_ntp.c_str());
```
[the remark this answers is not in the document]
Comment (HuuPhuc25) [reply applications/web_server/src/time_ntp_js.cpp]: Let for me example?

[the remark this answers is not in the document]
Comment (quoctri1) [reply applications/web_server/src/time_ntp_js.cpp]: I mean printf or syslog, but you shouldn't remove if you need it

**Pull request merged**